src/org/scada_lts/dao/report/ReportDAO.java

```diff
@@ -212,4 +246,26 @@ public void delete(int id) {
 
 		DAO.getInstance().getJdbcTemp().update(REPORT_DELETE, new Object[]{id});
 	}
+
+	public String joinAnd(List<String> conditions) {
+		StringBuilder result = new StringBuilder();
+		result.append(" 1 ");
+		for (String c:conditions) {
+			result.append(" AND (");
+			result.append(c);
+			result.append(") ");
+		}
+		return result.toString();
+	}
+
+	public String joinOr(List<String> conditions) {
+		StringBuilder result = new StringBuilder();
+		result.append(" 0 ");
+		for (String c:conditions) {
+			result.append(" OR (");
+			result.append(c);
+			result.append(") ");
+		}
+		return result.toString();
+	}
```

Comment:
Really nice methods! What do you think about lifting them up to more global class? Maybe we should consider the SqlUtils.java static class where you could move your methods and make them accessible from other places?